### ISSUE TITLE
moonrise / moonset.

### ISSUE THREAD

**Issue by extremeshok** [rejuvenate/lovelace-horizon-card#63]
To me the values make no sense, currently what is shown

Sunrise : 7:49
Sunset :  17:59
---
Moonrise : 5:05
Moonset : 15:07

Surely the moonrise and moonset need to be swapped around.

Basically I would like to know what time the moon will rise and what time it will set.

**Comment by avataar** [comment]
No, what you see is correct. Moonrise means "Moon goes above the horizon" and moonset means "Moon goes below the horizon". There are times of the year when the Moon is indeed above the horizon during the day (as happens to be now). Sometimes you can even see the Moon during the day if the conditions are right - position of the Sun, clouds, etc.

**Comment by extremeshok** [comment]
at 21h00 would the moon be visible or not ?

**Comment by avataar** [comment]
Obviously not, since at 15:07 it will go below the horizon. I suggest you play with the setting `now` in the card config, it allows you to set a fixed time (the Sun and the Moon won't move but will show that exact time), for example (adjust the timezone +0300 to match yours):

Compare this:
```yaml
now: 2023-07-14T21:00:00+0300
```

To this:
```yaml
now: 2023-07-30T21:00:00+0300
```

**Comment by ThomDietrich** [comment]
Hey @extremeshok, was your question answered? Perhaps you want to propose an addition to the text shown here? https://github.com/rejuvenate/lovelace-horizon-card#visibility-fields
Otherwise please close the ticket. Thanks!

**Comment by ThomDietrich** [comment]
Thanks for asking! This one is actually expected behaviour: unlike the sun, the moon regularly rises and sets during daylight hours, so a moonrise "in the middle of the day" is correct — the times follow the real lunar schedule for your location, not the day/night cycle.

Nothing to fix here, so I'll close this, but happy to reopen if something still looks off. Cheers!
